packages/fxa-auth-server/lib/payments/capability.ts

```diff
@@ -237,7 +240,7 @@ export class CapabilityService {
     uid: string
   ): Promise<ClientIdCapabilityMap> {
     const subscribedPrices = await this.subscribedPriceIds(uid);
-    return this.priceIdsToClientCapabilities(subscribedPrices);
+    return this.planIdsToClientCapabilities(subscribedPrices);
```

Comment:
Not blocking. Is there a reason for the name change? priceIdsToClientCapabilities seems to be the more correct naming convention.

Reply:
It was updated in [PR#15787](https://github.com/mozilla/fxa/pull/15787/files#diff-fff1d12858738e7446b8e99b827296c0714f16e9ce1e3f115743467f7ba04ace) in the new CapabilityManager. To keep it consistent, I updated all of the old references as explained in Slack. I initially kept them but it was confusing to read both terms.

Reply:
Sweet sounds good!